tools/tests.props

```diff
@@ -0,0 +1,32 @@
+<Project>
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup Label="DotnetTest">
+    <TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>
+    <TestingPlatformShowTestsFailure>true</TestingPlatformShowTestsFailure>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Testing.Extensions.CodeCoverage" />
+    <PackageReference Include="NUnit" />
+    <PackageReference Include="NUnit.Analyzers">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="NUnit3TestAdapter" />
+  </ItemGroup>
+
+  <!-- Must be removed when https://youtrack.jetbrains.com/issue/RSRP-498837 is fixed -->
+  <ItemGroup Condition="$(ContinuousIntegrationBuild) != 'true' AND $(BuildingInsideVisualStudio) != 'true'">
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+  </ItemGroup>
+  <!-- Enable `NUnitRunner` only on Visual Studio or CI -->
+  <PropertyGroup Condition="$(ContinuousIntegrationBuild) == 'true' OR $(BuildingInsideVisualStudio) == 'true'">
+    <EnableNUnitRunner>true</EnableNUnitRunner>
```

Comment:
- `EnableNUnitRunner` must be unconditional
- I think it's better not to have `tests.props` copy. Either link the one in `src/Nethermind` (since we already link projects from there) or simply move the stuff to the `.csproj` file, as it's the only one now.